migration/20171208-add-lane-size.sql

```diff
@@ -1 +1,9 @@
-alter table lanes add column size integer not null default 0;
+DO $$ 
+    BEGIN
+        BEGIN
+            ALTER TABLE lanes ADD COLUMN size integer not null default 0;
+        EXCEPTION
+            WHEN duplicate_column THEN RAISE NOTICE 'column lanes.size already exists, not creating it.';
+        END;
+    END $$;
+
```

Comment:
I've had a lot of problems 'migrating' sites that already have `lanes.size` by the time this script gets to them. I think it's because these sites don't have the `lanes` table to begin with, and it's created with `size` intact.